src/View.Uno.WinUI/Converters/PrettyDateConverter/PrettyDateFormatter.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Nventive.View.Converters.PrettyDateConverter.Resources;
+#if !WINUI && !__ANDROID__ && !__IOS__ && !__WASM__
+using Uno.Localisation;
+#else
+using CultureInfo = System.String;
+#endif
+namespace Nventive.View.Converters
+{
+	public interface IPrettyDateFormatter
+	{
+		string Format(TimeSpan elapsed, CultureInfo culture, PrettyDateMode mode = PrettyDateMode.Past);
+	}
+
+	public enum PrettyDateMode
+	{
+		Past = 0,
+		Neutral = 1
+	}
+
+	public class PrettyDateFormatter : IPrettyDateFormatter
+	{
+		private const string separator = " ";
+		public double NumberOfHoursBeforeDayFormatting { get; set; }
+
+		public PrettyDateFormatter()
+		{
+			NumberOfHoursBeforeDayFormatting = 48.0d; //To avoid breaking changes
+		}
+
+		public string Format(TimeSpan elapsed, CultureInfo culture, PrettyDateMode mode = PrettyDateMode.Past)
+		{
+			if (elapsed.TotalSeconds < 1)
+			{
+				elapsed = TimeSpan.Zero;
+			}
+
+			//if (dateTime.Age(now) >= 1)
```

Comment:
Why is this commented out?

Reply:
Can't say... This is a verbatim copy of the UWP implementation.